src/Accounts/Accounts/ChangeLog.md

```diff
@@ -29,6 +29,10 @@
 
 ## Version 1.3.0
 * Update to latest version of ClientRuntime
+* Azure automation update management change to support the following new features :
```

Comment:
@jemex the text you are adding to this file should go under the `## Upcoming Release` header -- looks like this moved down when the last release occurred